README.md

```diff
@@ -1690,6 +1690,7 @@ Update Time, five active automations, webhooks.
    * [Aptabase](https://aptabase.com) — Open Source, Privacy-Friendly, and Simple Analytics for Mobile and Desktop Apps. SDKs for Swift, Kotlin, React Native, Flutter, Electron, and many others. Free for up to 20,000 events per month.
    * [Trackingplan](https://www.trackingplan.com/) - Automatically detect digital analytics, marketing data and pixels issues, maintain up-to-date tracking plans, and foster seamless collaboration. Deploy it to your production environment with real traffic or add analytics coverage to your regression tests without writing code.
    * [LogSpot](https://logspot.io) - Full unified web and product analytics platform, including embeddable analytics widgets and automated robots (slack, telegram, and webhooks). Free plan includes 10,000 events per month.
+   * [AppFit](https://appfit.io) - AppFit is a comprehensive analytics and product management tool designed to facilitate seamless, cross-platform management of analytics and product updates. Free plan includes manual metrics and product journal.
```

Comment:
I admit to not having deeply studied this app so please correct me if I am wrong, but the free plan requires you to manually enter metrics - for a procut whose purpose is to collect real time metrics?

That's just does not seem useful in general but specifically does not allow devs to really build integrations for their customers without incuring costs and seeing the solutions in actual use - roughly thats the purpose of this list.

Reply:
Thanks, note though, I am not looking for some token amount just to cross the line between have something free or not.  Free SERVICES have to actually be a service that has use and a purpose.